pipelines/fbc-release/README.md

```diff
@@ -30,6 +30,10 @@ Tekton release pipeline to interact with FBC Pipeline
 
 ## Changelog
 
+### Changes since 0.22.0
```

Comment:
We don't have standards around this yet since the new versioning scheme is new, but I wonder if this should be 0.22.1 or 0.23.0. We have to decide as a team what constitutes bumping which version numbers

Reply:
FYI @redhat-appstudio/release-team

Reply:
Already decided https://semver.org/

```
Given a version number MAJOR.MINOR.PATCH, increment the:

1. MAJOR version when you make incompatible API changes
2. MINOR version when you add functionality in a backward compatible manner
3. PATCH version when you make backward compatible bug fixes
```